src/public/js/rs.js

```diff
@@ -1,16 +1,33 @@
 'use strict';
 var RS = (function() {
-  // (actually ROT-13 right now, as a dummy example. We can replace w/ RS)
-  var ROT = 13;
+  // Using a RS with 256 symbol size 255 symobls in a codeword and 245 message symbols.
+  // n-k = 10 code symbols. Can correct up to 5 symbol errors in each 255 symbol block
+  var n = 255;
+  var k = 245;
 
   function encode(msg) {
-    // dummy cipher for now -- advances each letter in message by ROT
-    return UTIL.shift(msg, ROT);
+    // Does Reed Solomon decoding on k length message block of symbols.
+    // Cast msg to Uint8Array
+    // If msg is not length k then pad with 0s.
+    var rsMsg = new Uint8Array(k);
+    for (var i = 0; i < msg.length; i++) {
+      rsMsg[i] = msg[i].charCodeAt();
+    }
+    return encodeRSBlock(rsMsg, n, k);
   }
 
   function decode(msg) {
-    // dummy cipher for now -- advances each letter in message by -ROT
-    return UTIL.shift(msg, -ROT);
```

Comment:
can you go clear the shift method out of the utils file if it's still there? irrelevant now